$NetBSD: patch-aa,v 1.3 2011/06/02 19:16:47 othyro Exp $

kill /opt in $prefix handling, or not works expected if ${LOCALBASE} start with
/opt.

--- configure.orig	2010-10-31 22:10:20.000000000 +0000
+++ configure
@@ -2661,30 +2661,6 @@ if test "${with_lsb+set}" = set; then :
 fi
 
 
-case "$prefix" in
-/opt/*)
-    if test "$datadir" == '${prefix}/share' ; then
-        if test "$mandir" == '${datadir}/man' ; then
-            mandir='${prefix}/man' ; fi
-        if test "$infodir" == '${datadir}/info' ; then
-            infodir='${prefix}/info' ; fi
-        datadir='${prefix}'
-    fi
-    if test "$sysconfdir" == '${prefix}/etc' ; then
-        sysconfdir=/etc${prefix} ; fi
-    if test "$localstatedir" == '${prefix}/var' ; then
-        localstatedir=/var${prefix} ; fi
-    if test "$includedir" == '${prefix}/include' ; then
-        ccincludedir="$includedir" ; fi
-
-
-
-
-
-
-    ;;
-esac
-
 if test -z "$ccincludedir" ; then
     case "$includedir" in
     */lib/*)
